src/components/DeploymentQuickRunOverflowMenuItem.vue

```diff
@@ -13,29 +13,40 @@
   import ToastFlowRunCreate from '@/components/ToastFlowRunCreate.vue'
   import { useWorkspaceApi, useWorkspaceRoutes } from '@/compositions'
   import { localization } from '@/localization'
+  import { Deployment } from '@/models'
 
   const props = defineProps<{
-    deploymentId: string,
+    deployment: Deployment,
+    openModal: () => void,
   }>()
 
   const api = useWorkspaceApi()
   const router = useRouter()
   const routes = useWorkspaceRoutes()
 
   const run = async (): Promise<void> => {
-    try {
-      const flowRun = await api.deployments.createDeploymentFlowRun(props.deploymentId, {
-        state: {
-          type: 'scheduled',
-          message: 'Run from the Prefect UI with defaults',
-        },
-      })
+    if (props.deployment.parameterOpenApiSchema.required) {
```

Comment:
@bsignoret this gave me a bit of pause just from a readability perspective. Suggestion:
```suggestion
    if (props.deployment.parameterOpenApiSchema.required?.length > 0) {
```

This will ensure 1) the key exists and 2) there are required properties in the list; your implementation works but imo requires too much thought into how truthiness works in JS